### PR TITLE
ui/ux updates

### PR DESCRIPTION
**Pull request by adamraider**

### CODE REVIEW

File: public/storage [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/adamraider/tokenly/artist-music-manager/storage/app/public
```
Comment (deweller) [inline public/storage]: @adamraider - This is just an artifact we can delete, correct?

Comment (adamraider) [inline public/storage]: yes

Comment (deweller) [inline public/storage]: Same with the node_modules/.bin files later in the commit too.  Right?

Comment (adamraider) [inline public/storage]: yeah those shoudlve been gitignored. not sure what happened. sorry about that

Comment (deweller) [inline public/storage]: Looks like they were deleted.  I thought they were being added.  All good.

**Pull request merged**